src/main/python/resources/annotaters/vep/template_sift_polyphen.py

```diff
@@ -0,0 +1,30 @@
+from src.main.python.resources.annotaters.vep.template import Template
+import pandas as pd
+
+
+class TemplateSiftPolyPhen(Template):
+    def __init__(self, name='Template', usable=False):
+        super(TemplateSiftPolyPhen, self).__init__(
+            name=name,
+            usable=usable
+        )
+
+    @property
+    def columns(self):
+        return [None, None]
+
+    @property
+    def val_col(self):
+        return self.columns[1]
+
+    def process(self, dataframe: pd.DataFrame):
+        dataframe[self.columns] = dataframe[self.name].str.split(
+            '(',
+            expand=True
+        )
+        dataframe[self.val_col] = dataframe[self.val_col].str.split(
+            ')',
+            expand=True
+        )[0]
+        dataframe[self.val_col] = dataframe[self.val_col].astype(float)
+        return dataframe
```

Comment:
> Why make all these adjustments within the dataframe itself instead of replacing one value with another?

First, regex becomes unreadable very fast (compared to a simple `str.replace('-?', '', regex=False)`). Secondly, I try to keep as much within Pandas as possible, since Pandas is an extension of Numpy, which functions are fully written in `C`. So it's for performance reasons.